docs/source/llms/index.rst

```diff
@@ -24,15 +24,15 @@ Serving as a unified interface, the `MLflow Deployments Server <deployments/inde
 `OpenAI <https://openai.com/>`_, `MosaicML <https://www.mosaicml.com/>`_, `Cohere <https://cohere.com/>`_, `Anthropic <https://www.anthropic.com/>`_, 
 `PaLM 2 <https://ai.google/discover/palm2/>`_, `AWS Bedrock <https://aws.amazon.com/bedrock/>`_, and `AI21 Labs <https://www.ai21.com/>`_. 
 
-In addition to supporting the most popular SaaS LLM providers, the AI Gateway provides an integration to MLflow model serving, allowing you to serve your 
+In addition to supporting the most popular SaaS LLM providers, the MLflow Deployments Server provides an integration to MLflow model serving, allowing you to serve your 
```

Comment:
```suggestion
In addition to supporting the most popular SaaS LLM providers, the MLflow Deployments Server (previously called "AI Gateway") provides an integration to MLflow model serving, allowing you to serve your 
```

To help ppl already used the feature? WDYT?

Reply:
updated!